packages/web_drivers/lib/chrome_driver_installer.dart

```diff
@@ -100,6 +110,23 @@ class ChromeDriverInstaller {
     return int.parse(versionNo);
   }
 
+  /// Find Google Chrome App on Mac.
+  Future<String> findChromeExecutableOnMac() async {
+    final io.ProcessResult processResult = await io.Process.run('find',
+        <String>['/Applications', '-type', 'd', '-name', '*Chrome.app']);
```

Comment:
Do we need to recursively scan all sub-directories? It could be quite slow. Perhaps use `-maxdepth 1`? (the `listSync` approach above also fixes this)

Reply:
resolving the comment since the previous suggestion applied.